templates/components/products/carousel.html

```diff
@@ -4,13 +4,29 @@
         "dots": true,
         "infinite": false,
         "mobileFirst": true,
-        "slidesToShow": {{ columns }},
-        "slidesToScroll": 3
+        "slidesToShow": 2,
+        "slidesToScroll": 2,
+        "responsive": [
+            {
+                "breakpoint": 800,
+                "settings": {
```

Comment:
Are these the standard breakpoints we use platform wide?

Reply:
Confirmed these are in alignment

Reply:
Yes, 800px and 550px are breakpoints for cornerstone.